package.json

```diff
@@ -159,7 +159,7 @@
     "jsdom": "^12.2.0",
     "jsonld": "^5.2.0",
     "jsonlint-mod": "^1.7.6",
-    "lighthouse-plugin-publisher-ads": "^1.5.6",
+    "lighthouse-plugin-publisher-ads": "https://gitpkg.now.sh/googleads/publisher-ads-lighthouse-plugin/lighthouse-plugin-publisher-ads?main",
```

Comment:
Could we publish an `@next`/`@esm`/`@whatever` version instead?

Reply:
I'm not a fan of the extra process / dependency this would add on an outside team. There's also may be a bit more back/forth to be expected here before 10.0 is released, so I want to wait until we're ready to actual do that cut. Ugh, two-way deps are annoying. This will go back to an actual published version by 10.0.

Reply:
It's suboptimal for pubads to require googler reviews and MFA for publishing, and then we pipe it through someone's random git packaging service :) This should be long gone before we bundle the result and ship it anywhere.

Reply:
> This should be long gone before we bundle the result and ship it anywhere.

That's the plan.